root/treeformula/scan/Makefile

```diff
@@ -62,6 +62,9 @@ endif
 # By default the TestDiff variable is set to compare the .log and .ref files
 # To run a diff and ignore whitespacing use $(TestDiffW)
 
+LoadDim: LoadDim.log
```

Comment:
The change in the `Makefile` can also be replaced by 'just' renaming the script to have an `exec` prefix: `execLoadDim.C` ;  The function should (of course) be also renamed and be made to `return 0` upon success for good measure.  The reference file should also be renamed be `execLoadDim.ref`.

Reply:
Fixed